util/testutil/integration/containerd.go

```diff
@@ -202,7 +205,7 @@ disabled_plugins = ["cri"]
 			return nil, nil, err
 		}
 		buildkitdArgs = append([]string{"sudo", "-u", fmt.Sprintf("#%d", c.uid), "-i", "--", "exec",
-			"nsenter", "-U", "--preserve-credentials", "-m", "-t", fmt.Sprintf("%d", pid)},
+			"nsenter", "-U", "--preserve-credentials", "-m", "-n", "-t", fmt.Sprintf("%d", pid)},
 			append(buildkitdArgs, "--containerd-worker-snapshotter=native")...)
 	}
```

Comment:
Why not use `containerd-rootless-setuptool.sh nsenter` ?

Reply:
This is because `containerd-rootless-setuptool.sh` doesn't seem to support a customized location of rootlesskit state dir.